### DISCUSSION TITLE
Download to SD card, Android

### DISCUSSION THREAD

**Discussion by jelabarre59** [localsend/localsend#1183] [Discussion (English, International)]
I can't find how to set LocalSend's downloads directory to my SD card (Android app).  It seems to only see the internal storage.  Didn't find any existing comments on it here.

**Comment by Tienisto** [comment]
Go to settings, there you can set the destination directory. Be aware that on Android 10 or older, you need LocalSend 1.14.0

**Comment by jelabarre59** [comment]
That was where I had been trying to set it.  But eventually I found where the SD card was hidden in the directory selector (don't know how I didn't see the card show up when I was looking before, maybe mis-remembering the volume name)
